src/masonite/routes/HTTPRoute.py

```diff
@@ -141,9 +141,6 @@ def get_response(self, app=None):
                 # *self.request.url_params.values() TODO
             )
 
-            if hasattr(response, "rendered_template"):
-                response = response.rendered_template
-
```

Comment:
Yeah we don't need this anymore cuss we added get_response() in the view class which gets called in the response class. This is an example of some old legacy code still in the codebase